<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0); 
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0,0);</string>
       </property>
       <property name="text">
        <string>Cadastro De Clientes</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>311</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 47, 47);
color: rgb(255, 255, 255);
border-radius:10;</string>
       </property>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>62</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(85, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Nome :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nome">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>44</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(85, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>CPF :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cpf">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>58</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(85, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>85</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(85, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Telefone :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="telefone">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>60</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(85, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="senha">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cadastrar_cliente">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>330</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 150, 35);</string>
        </property>
        <property name="text">
         <string>Cadastrar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_voltar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);</string>
        </property>
        <property name="text">
         <string>Voltar</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>10</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0,0);</string>
       </property>
       <property name="text">
        <string>Clientes Cadastrados</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>90</y>
         <width>1091</width>
         <height>731</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
